library/Magicbane.hs

```diff
@@ -38,6 +36,15 @@ import           Magicbane.Metrics as X
 import           Magicbane.Validation as X
 import           Magicbane.HTTPClient as X
 import           Magicbane.Util as X
+-- replacing ClassyPrelude
```

Comment:
- please don't add the "replacing ClassyPrelude" comment
- no need to reexport everything that ClassyPrelude offered! only things used in Magicbane itself!!
- move these imports to the top, they shouldn't be under `Magicbane.*` imports

(same in other files)

Reply:
Comment was just FYI to highlight changes, I expected it to be going away before merge. :)

The only thing I explicitly held on to is the Concurrent/Async stuff, because per your comments on master, there are combinations that don't work with ExceptT. Seemed worth asserting one that does. I guess this could just be clarified in docs, tho... then `yieldThread` goes away too (Classy renames that to avoid namespace conflict with conduit/pipes, I was just preserving the whole conceptual group as-is).

Reply:
diff itself is a highlight of changes :)

Oh okay I guess that makes sense, maybe keep `yieldThread` but document why it's there